apps/src/aichat/types.ts

```diff
@@ -27,3 +29,9 @@ export enum Status {
   INAPPROPRIATE = 'inappropriate',
   UNKNOWN = 'unknown',
 }
+
+export interface AichatLevelProperties extends LevelProperties {
+  systemPrompt?: string;
```

Comment:
Do we want to go ahead and make any of these required?

Reply:
I'd like to touch base with curriculum first so include in follow-up - thanks!